lib/dnsimple/client/registrar.rb

```diff
@@ -41,6 +41,27 @@ def register(account_id, domain_name, attributes = {}, options = {})
         Dnsimple::Response.new(response, Struct::Domain.new(response["data"]))
       end
 
+      # Renews a domain.
+      #
+      # @see https://developer.dnsimple.com/v2/registrar/#renew
+      #
+      # @example Renew example.com for 3 years:
+      #   client.registrar.renew(1010, "example.com", period: 3)
+      #
+      # @param  [Fixnum] account_id the account ID
+      # @param  [#to_s] domain_name The domain name to register.
+      # @param  [Hash] attributes
```

Comment:
Can you clarify the renaming bit? My guess is that you only meant to rename `renew` to `renew_domain` and let every other registrar action's name that is domain related untouched.

In other words:
- Registrar#check_domain
- Registrar#register_domain
- Registrar#transfer_domain
- Registrar#transfer_out_domain
- Registrar#renew_domain

VS.

- Registrar#check_domain
- Registrar#register
- Registrar#transfer
- Registrar#transfer_out
- Registrar#renew_domain

(My vote goes for 1. for consistency).